scripts/extras/foglamp.sudoers

```diff
@@ -1,2 +1,3 @@
-%sudo ALL=(ALL) NOPASSWD: /usr/bin/apt-get -y update, /usr/bin/apt-get -y install foglamp
-
+%sudo ALL=(ALL) NOPASSWD: /usr/bin/apt-get -y update
+%sudo ALL=(ALL) NOPASSWD: /usr/bin/apt-get -y install foglamp
```

Comment:
Use `apt` instead of `apt-get` as apt is more structured and provides you with necessary options needed to manage packages.

Reply:
Let us take up this improvement separately as it really belongs to foglamp-pk repo and it will need thorough testing with deb packages.

Reply:
We could drop tis file from the PR altogether now (and update the description of the PR), since we are to unsung sudo any more for this.